handlers/PRHandler.go

```diff
@@ -1,10 +1,10 @@
 package handlers
```

Comment:
- **Error Messages Formatting:** Ensure consistent formatting of error messages in `ctx.JSON` calls. For instance, in `StorePRData`, you have a colon and space inconsistently placed in error messages like `"error parsing request body. error:"` and `"error fetching pr changes", "error:": err.Error()`. Choose one format for clarity.
- **Error Handling:** Avoid leaking internal logic, such as error structures, directly in JSON responses. Provide user-friendly error messages instead.
- **Magic Value:** The `http.StatusOK` is used as both a value and a string in `gin.H{"status": http.StatusOK}` in the last line of `StorePRData`. Ensure consistent representation.